### DISCUSSION TITLE
Unable to find component: [livewire.datatable.pay-slip-table]

### DISCUSSION THREAD

**Discussion by nshibu** [rappasoft/laravel-livewire-tables#785] [Q&A]
What is the problem?

**Comment by inmanturbo** [comment]
This looks like a namespace error. check that you don't have any typos in your namespace, class name, or file name. Then check that you are using correct dot kebab to render. For instance if the fully qualified class name is `App\Http\Livewire\DataTable\PaySlipTable`. You would render it with
```blade
<livewire:data-table.pay-slip-table />
```
